### PR TITLE
Added channels logos for it.m3u

### PR DESCRIPTION
**Pull request by YacineBoussoufa**
Added channels logos, and changed HSE 24 Italia in GM24 (they have changed name a couple of days ago)

### CODE REVIEW

File: channels/it.m3u [hunk 1]
```diff
@@ -25,23 +25,23 @@ http://live.sloode.com:1935/canale21/live2/playlist.m3u8
...
...7bSPgcwYUXVdkWUFtz9zQ5aa9jxQDQzfnA29O00eFI3WZ+i21a66Z+vIXHmwBrDRYjByImYCnY+c3lzYyBE9o131rofEF1/ujburkEZuMFsszqeZJFa5SyGGkivmDmwghbogln45mOhEovZtF1+Muq2RcdqXImHS6droWOpwLfIwB0+Edu+Ze8j5HRbSAGrbKXRGxT8ZpadI1RYaK8m8YVpce1oY/ths2yXKlFkWYaHLWQ8K4oq6ZJdbXjjs25sEoi2dqgLeoW6LxpVMvScKtc0lPlKgMKll7FLasxv1KXbZf8CRQXOB3YbFBs3gdW2f1JY2zm6ODjA/kD10dOjs7XCZNCPTQAZW/XKIhVm+tM4SIsFJdUE3Vkyu6LX9jeDlIjG6t28HQk/MQyxpaxdrb8QrgK2ahctg1f3fYV5y+VAzWBnQMROH3wYNtOWKsW9HVSvxU+wD9mXcJJuCpqCnf1gdJOxb6ECpyIHjhv5cvSP7iAeGnbCVbALgSzPeg/qOloIouwjFe9p2tt/OXbKluisFPfZtILTB8KC0u4zFnNnAMP0yCmZAj/VE/LTqNjB+Kg7mM7wROwURbRV07TFrGbbk7VRrWpXDIrrPbgTsGdadufwO+rpQ+I+EfoEWAicence4afbRqXpW8P6UaYwedCeLs1FbzooFtbOLsZq2gGtwmjckRP4manmpl8vKmi1fTuEPgYtlc2i7Ox1z6tpg8/dYS7GTE0rXcpXSkF1q5zPp9Ppbd/3y07Jzm8aJjNBZuJeQbH6UCMMRqF6B/P3K+u/T0EKd8EVmwaSabe+W7bMJPSJxnay/zTum4KRXDRKRfQfo4eqYDwKUgXtcLPS8dcLYqOWFGjGP7yo9MVA3z/cCEoROrOIncRMV2iTpJE4RjEEfhJiL5xlOGRs2s4480UDFPcFwlkkvQcfTu1qpWJ5oikMuvyAlGgzh+zGPA/mTwUXyvU8Y7Oz1aq/HyBRfmKv6O8GxGacOaGk+5PM0+M8ePFpsGcAqUWfQ9AqpmlZtl1uGsdH7+NYlK6DUfM9aIbPQArt5izaNLEslFFqZo7WXw+90u7DGKplr7axfpR5VWzHQRwFm8Sk4raePzIoctKD8wcgxX/DNbDcvedbj+Xf8APB/wcZ4zHoAt/5jAAAAABJRU5ErkJggg==" group-title="",Carina TV (480p)
```
Comment (freearhey) [inline channels/it.m3u:32]: Please do not use base64 encoded images here.

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/it.m3u (hunk 1 above)
Comment (YacineBoussoufa) [reply channels/it.m3u]: Removed

File: channels/it.m3u [hunk 2]
```diff
@@ -91,17 +91,17 @@ https://sb.top-ix.org/streamtech/tvlive1/playlist.m3u8
...
...apNEOEykMEjRDhIpDtceU9ZRvMqlwGLjynrKHJlJLgS15HGespWy6iOZHHjPWUbzGoxBpPKespbzLSXAbTP2j+Yo3mOlwDTd9o/mKN5hS4Bpn7R/MUb7BxXANN32j+Yo32LdXANN32nfmKN9hS4Bpu+078xRvMKXAC48p/MU7YUuBHnPWUBS4B5z1lIdLgF+k9ZQOlwC/SesoClwC/SesoFS4BfpPWUWG6uAAnlPWU7DdXACTynrKLYbq4BfpPWUWxNLgKXdJ6ylYt1cBST/ANKe8yHFBpnlPWU95kOKFe48p6ynZDijG4/9ulZO6l1CFMTFKZDEKEZsQpkMQoJFQQPx+coY0O1ItDNKDVDAoKQwKRaHBSKQwSLQ6BokILTJSLTGBQOybqSrBUKwQOwQILoALoGF0CC6BoLoHYICwQFggLBAWCAsECsglArFIQTZBTQmRdBDFJTRDFcgkxlNEMUpkMUoIYhTIYrk0QxEyBjv+cqRpjAoLQ4QaIYJFoYJFodpSLHBSGiWuTLGukNMm6C0ybpDTC6B2TdAWF0BYXQFhdA7C6BWF0DsLoFYXQFhdAWCAsLoCwugLC6AsglAWRdAWTdBNkIE2I4pkNikpkkOKaJMZKCSCmjNiFMgQoIYrkECFAhidp8JVCQwKktDhBaYwSLTGCRomMCgpMcJDTJCCkxgUqLskFBVk3QOyboHYXSFZF0h2TdAWF0wsLpBZF0BYXTCyUBYXTCyEBZN0hWF0gsLpjsLoCyLoE2TdNE2IXJ0JkXQSKSmTYhKCWxSiiLFKaJbFJTIYpKCGY3FBDIVEEnfPhKATJapZaY4QWmMCgtMcFJlpk3SKTGBQUmMHILskFIdjByB2F0h2TdFDsLooLC6KCwuigsLooLC6KCwuigsLooLC6KCwuigsLooLC6KCwRQWF0UFhdFBZBKdBYXQTZBKaFYpcgViEpktgSglsQlBDZF0EiOKZDYpKaIbEKZFkXQTZJO3zlAkyQgtMYFTRSYwKC0xgUFpjAoLTGBSGmTdBSZIKRdkgoEmSCkOyboHYXQOwugLC6AsLoCwugLC6AsLoCwugLC6AsLoCwugLC6AsLoCw0kBYaSBWQXJisW6BWRdBNkEoFYpKZDZBKCbEJTJbEKCLFJTRDZBKZDYl0xWf/Z" group-title="",Radio Capital TV (360p)
```
Comment (freearhey) [inline channels/it.m3u:100]: also base64

File: channels/it.m3u [hunk 3]
```diff
@@ -91,17 +91,17 @@ https://sb.top-ix.org/streamtech/tvlive1/playlist.m3u8
...
...aGapNEOEykMEjRDhIpDtceU9ZRvMqlwGLjynrKHJlJLgS15HGespWy6iOZHHjPWUbzGoxBpPKespbzLSXAbTP2j+Yo3mOlwDTd9o/mKN5hS4Bpn7R/MUb7BxXANN32j+Yo32LdXANN32nfmKN9hS4Bpu+078xRvMKXAC48p/MU7YUuBHnPWUBS4B5z1lIdLgF+k9ZQOlwC/SesoClwC/SesoFS4BfpPWUWG6uAAnlPWU7DdXACTynrKLYbq4BfpPWUWxNLgKXdJ6ylYt1cBST/ANKe8yHFBpnlPWU95kOKFe48p6ynZDijG4/9ulZO6l1CFMTFKZDEKEZsQpkMQoJFQQPx+coY0O1ItDNKDVDAoKQwKRaHBSKQwSLQ6BokILTJSLTGBQOybqSrBUKwQOwQILoALoGF0CC6BoLoHYICwQFggLBAWCAsECsglArFIQTZBTQmRdBDFJTRDFcgkxlNEMUpkMUoIYhTIYrk0QxEyBjv+cqRpjAoLQ4QaIYJFoYJFodpSLHBSGiWuTLGukNMm6C0ybpDTC6B2TdAWF0BYXQFhdA7C6BWF0DsLoFYXQFhdAWCAsLoCwugLC6AsglAWRdAWTdBNkIE2I4pkNikpkkOKaJMZKCSCmjNiFMgQoIYrkECFAhidp8JVCQwKktDhBaYwSLTGCRomMCgpMcJDTJCCkxgUqLskFBVk3QOyboHYXSFZF0h2TdAWF0wsLpBZF0BYXTCyUBYXTCyEBZN0hWF0gsLpjsLoCyLoE2TdNE2IXJ0JkXQSKSmTYhKCWxSiiLFKaJbFJTIYpKCGY3FBDIVEEnfPhKATJapZaY4QWmMCgtMcFJlpk3SKTGBQUmMHILskFIdjByB2F0h2TdFDsLooLC6KCwuigsLooLC6KCwuigsLooLC6KCwuigsLooLC6KCwRQWF0UFhdFBZBKdBYXQTZBKaFYpcgViEpktgSglsQlBDZF0EiOKZDYpKaIbEKZFkXQTZJO3zlAkyQgtMYFTRSYwKC0xgUFpjAoLTGBSGmTdBSZIKRdkgoEmSCkOyboHYXQOwugLC6AsLoCwugLC6AsLoCwugLC6AsLoCwugLC6AsLoCw0kBYaSBWQXJisW6BWRdBNkEoFYpKZDZBKCbEJTJbEKCLFJTRDZBKZDYl0xWf/Z" group-title="Music",Radio Capital TV
```
Comment (freearhey) [inline channels/it.m3u:102]: also base64

File: channels/it.m3u [hunk 4]
```diff
@@ -91,17 +91,17 @@ https://sb.top-ix.org/streamtech/tvlive1/playlist.m3u8
...
...apNEOEykMEjRDhIpDtceU9ZRvMqlwGLjynrKHJlJLgS15HGespWy6iOZHHjPWUbzGoxBpPKespbzLSXAbTP2j+Yo3mOlwDTd9o/mKN5hS4Bpn7R/MUb7BxXANN32j+Yo32LdXANN32nfmKN9hS4Bpu+078xRvMKXAC48p/MU7YUuBHnPWUBS4B5z1lIdLgF+k9ZQOlwC/SesoClwC/SesoFS4BfpPWUWG6uAAnlPWU7DdXACTynrKLYbq4BfpPWUWxNLgKXdJ6ylYt1cBST/ANKe8yHFBpnlPWU95kOKFe48p6ynZDijG4/9ulZO6l1CFMTFKZDEKEZsQpkMQoJFQQPx+coY0O1ItDNKDVDAoKQwKRaHBSKQwSLQ6BokILTJSLTGBQOybqSrBUKwQOwQILoALoGF0CC6BoLoHYICwQFggLBAWCAsECsglArFIQTZBTQmRdBDFJTRDFcgkxlNEMUpkMUoIYhTIYrk0QxEyBjv+cqRpjAoLQ4QaIYJFoYJFodpSLHBSGiWuTLGukNMm6C0ybpDTC6B2TdAWF0BYXQFhdA7C6BWF0DsLoFYXQFhdAWCAsLoCwugLC6AsglAWRdAWTdBNkIE2I4pkNikpkkOKaJMZKCSCmjNiFMgQoIYrkECFAhidp8JVCQwKktDhBaYwSLTGCRomMCgpMcJDTJCCkxgUqLskFBVk3QOyboHYXSFZF0h2TdAWF0wsLpBZF0BYXTCyUBYXTCyEBZN0hWF0gsLpjsLoCyLoE2TdNE2IXJ0JkXQSKSmTYhKCWxSiiLFKaJbFJTIYpKCGY3FBDIVEEnfPhKATJapZaY4QWmMCgtMcFJlpk3SKTGBQUmMHILskFIdjByB2F0h2TdFDsLooLC6KCwuigsLooLC6KCwuigsLooLC6KCwuigsLooLC6KCwRQWF0UFhdFBZBKdBYXQTZBKaFYpcgViEpktgSglsQlBDZF0EiOKZDYpKaIbEKZFkXQTZJO3zlAkyQgtMYFTRSYwKC0xgUFpjAoLTGBSGmTdBSZIKRdkgoEmSCkOyboHYXQOwugLC6AsLoCwugLC6AsLoCwugLC6AsLoCwugLC6AsLoCw0kBYaSBWQXJisW6BWRdBNkEoFYpKZDZBKCbEJTJbEKCLFJTRDZBKZDYl0xWf/Z" group-title="",Radio Capital TV (360p)
```
Comment (freearhey) [inline channels/it.m3u:104]: also base64

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
@YacineBoussoufa Thanks!

**Comment by YacineBoussoufa** [comment]
> @YacineBoussoufa Thanks!

Sorry if I did some mistakes, this was my second pull request.

**Comment by freearhey** [comment]
@YacineBoussoufa don't worry.

**Pull request merged**